6-AdvancedScenarios/4-hybrid-SPA/App/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "hybrid-spa-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "cd client && npm run build",
```

Comment:
Can we add preinstall and prestart scripts here for the client app?

Reply:
done!